keep-libruntime/fixtures/return_1.wkld

```diff
@@ -0,0 +1,8 @@
+{
+  "modules": {
+    "return_1": [0,97,115,109,1,0,0,0,1,133,128,128,128,0,1,96,0,1,127,3,130,128,128,128,0,1,0,4,132,128,128,128,0,1,112,0,0,5,131,128,128,128,0,1,0,1,6,129,128,128,128,0,0,7,145,128,128,128,0,2,6,109,101,109,111,114,121,2,0,4,109,97,105,110,0,0,10,138,128,128,128,0,1,132,128,128,128,0,0,65,1,11]
+  },
+  "function_module":"return_1",
```

Comment:
space after `:` .. my eyes! :smile:

Reply:
I generated this and merely added newlines. I don't expect the tenant to write plain JSON. We can generate this from the command line even. I'm thinking of a CLI which takes an argument for all of the keys given here, i.e.
```console
`enxctl generate-workload --modules="./app.wasm,./somelib.wasm" --function_module="app" --function-name="main" --- "command_arg_arg-passed_to_function" --long-arg-passed="to_function".
```

In addition we can offer to parse a YAML or TOML for this purpose.